sql/views.sql

```diff
@@ -91,179 +106,190 @@ FROM _timescaledb_config.bgw_job j
-        WHERE srv.srvfdw = fdw.oid
-        AND fdw.fdwname = 'timescaledb_fdw') AS s;
+SELECT s.node_name,
+  s.owner,
+  s.options
+FROM (
+  SELECT srvname AS node_name,
+    srvowner::regrole::name AS owner,
+    srvoptions AS options
+  FROM pg_catalog.pg_foreign_server AS srv,
+    pg_catalog.pg_foreign_data_wrapper AS fdw
+  WHERE srv.srvfdw = fdw.oid
+    AND fdw.fdwname = 'timescaledb_fdw') AS s;
 
 -- chunks metadata view, shows information about the primary dimension column
 -- query plans with CTEs are not always optimized by PG. So use in-line
 -- tables.
-CREATE OR REPLACE VIEW  timescaledb_information.chunks
-AS
-SELECT 
-       hypertable_schema, hypertable_name,
-       schema_name as chunk_schema , chunk_name ,
-       primary_dimension, primary_dimension_type,
-       range_start, range_end,
-       integer_range_start as range_start_integer,
-       integer_range_end as range_end_integer,
-       is_compressed,
-       chunk_table_space as chunk_tablespace,
-       node_list as data_nodes
-       from
-(
-SELECT
-    ht.schema_name as hypertable_schema,
-    ht.table_name as hypertable_name,
-    srcch.schema_name as schema_name,
-    srcch.table_name as chunk_name,
-    dim.column_name as primary_dimension,
-    dim.column_type as primary_dimension_type,
-    row_number() over(partition by chcons.chunk_id order by dim.id) as chunk_dimension_num,
-    CASE
-        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
-               dim.column_type = 'TIMESTAMPTZ'::regtype OR
-               dim.column_type = 'DATE'::regtype )
-        THEN _timescaledb_internal.to_timestamp(dimsl.range_start)
-        ELSE NULL
-    END as range_start,
-    CASE
-         WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
-                dim.column_type = 'TIMESTAMPTZ'::regtype OR
-                dim.column_type = 'DATE'::regtype )
-         THEN _timescaledb_internal.to_timestamp(dimsl.range_end)
-         ELSE NULL
-    END as range_end,
-    CASE
-        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
-               dim.column_type = 'TIMESTAMPTZ'::regtype OR
-               dim.column_type = 'DATE'::regtype )
-        THEN NULL
-        ELSE dimsl.range_start
-    END as integer_range_start,
-    CASE
-        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
-               dim.column_type = 'TIMESTAMPTZ'::regtype OR
-               dim.column_type = 'DATE'::regtype )
-        THEN NULL
-        ELSE dimsl.range_end
-    END as integer_range_end,
-    CASE WHEN srcch.compressed_chunk_id is not null THEN true
-         ELSE false
-    END as is_compressed,
-    pgtab.spcname as chunk_table_space,
+
+CREATE OR REPLACE VIEW timescaledb_information.chunks AS
+SELECT hypertable_schema,
+  hypertable_name,
+  schema_name AS chunk_schema,
+  chunk_name,
+  primary_dimension,
+  primary_dimension_type,
+  range_start,
+  range_end,
+  integer_range_start AS range_start_integer,
```

Comment:
Also nit unrelated to formatting. A bit strange to first alias this to `integer_range_start` to then re-alias here to `range_start_integer`